code/game/machinery/photo_printer.dm

```diff
@@ -0,0 +1,21 @@
+/obj/machinery/photo_printer
+	name = "Photo Printer"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "pp_loaded"
+	density = 1
+	anchored = 1.0
+	use_power = 1
+	idle_power_usage = 30
+	active_power_usage = 200
+
+
+/obj/machinery/photo_printer/sec
+	name = "Security Photo Printer"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "sec_photo_printer"
+	density = 1
+	anchored = 1.0
+	use_power = 1
+	idle_power_usage = 30
+	active_power_usage = 200
```

Comment:
Objects inherit the parent object's variables, line 14 and 16-20 should be deleted as their variable values are identical to the parent.